### DISCUSSION TITLE
Which Image is best for a Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by danielrlutz** [dani-garcia/vaultwarden#1649] [Q&A] [answered]
Which image would be the best for a raspberry pi 4? Should I use 'vaultwarden/server:armv6' or is another image better for the raspberry pi 4?

**Comment by dani-garcia** [comment, marked as answer]
In most cases, using vaultwarden/server should select the most appropiate image for your architecture so it's the recommended way.

The armv6 tag exists for devices like the raspberry pi zero and it's only useful with some docker versions that can't pick the correct architecture to use.

**Reply by mzch** [reply]
Oh, I missed it. Thanks! :)
